Public/Src/FrontEnd/MsBuild/PipConstructor.cs

```diff
@@ -503,7 +503,8 @@ private void SetProcessEnvironmentVariables(IReadOnlyDictionary<string, string>
             processBuilder.AllowedSurvivingChildProcessNames = ReadOnlyArray<PathAtom>.FromWithoutCopy(
                 PathAtom.Create(m_context.StringTable, "mspdbsrv.exe"),
                 PathAtom.Create(m_context.StringTable, "vctip.exe"),
-                PathAtom.Create(m_context.StringTable, "conhost.exe"));
+                PathAtom.Create(m_context.StringTable, "conhost.exe"),
+                PathAtom.Create(m_context.StringTable, "VBCSCompiler.exe"));

```

Comment:
>VBCSCompiler [](start = 56, length = 12)

Small suggestion. It seems that this method is called for each and every pip.

It would be more performant if the the array would be created only once for the lifetime of the frontend and then reused.

The same holds true for a few other things like the arguments for logging etc.
Not needed in this PR. Just sharing to improve perf in the future.